### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by twilson-ay** [thoughtspot/cs_tools#189] [Q&A] [answered]
Hello. I am trying to install CS Tools, but keep getting an error that I cannot get around. When installing the tool through a command prompt, I receive the following error.

UnicodeDecodeError: 'utf-8' codec can't decode bytes in position 308-309: invalid continuation byte

Does anyone know what this means and how I can get around it?

**Comment by boonhapus** [comment]
Hey @twilson-ay , can you share the exact command you're using, and if there's any error log file that gets generated?

**Reply by twilson-ay** [reply]
I am running the command below from from the CS Tools tutorial (https://thoughtspot.github.io/cs_tools/tutorial/install/).

powershell -c "IRM https://raw.githubusercontent.com/thoughtspot/cs_tools/master/cs_tools/updater/_bootstrapper.py | python - --install"

I am running it in a command prompt on a Windows 10 machine. It does generate an error log.

**Reply by boonhapus** [reply]
Got it. I just ran this in my local windows test environments and it looks like it works there. Would you be able to meet briefly on Monday to look at it together? Please shoot me an email so we can find some time together.

**Reply by twilson-ay** [reply, marked as answer]
Sure. I can do that. I'll email you here in a few.
